packages/callback/README.md

```diff
@@ -0,0 +1,43 @@
+# Callback
+
+[![pub package](https://img.shields.io/pub/v/callback.svg)](https://pub.dartlang.org/packages/callback)
+
+A Flutter plugin that allows native callbacks to be registered and
+called from Dart side. You can use this plugin to communicate lifecycle events
+from Dart to native side. It is also useful for handling events that require
+dependencies between multiple plugins; a callback defined at app layer
+can orchestrate such logic.
+
```

Comment:
May be a good idea to document when you would use Callback rather than a straight MethodChannel.